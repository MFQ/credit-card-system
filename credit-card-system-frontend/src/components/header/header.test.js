import React from 'react';
import Header from './index';
import renderer from 'react-test-renderer';

test('Render Header', () => {
  const component = renderer.create(<Header />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});