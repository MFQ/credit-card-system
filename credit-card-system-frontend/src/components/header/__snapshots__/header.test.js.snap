// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Header 1`] = `
<nav
  className="navbar navbar-expand-lg navbar-light bg-light"
>
  <a
    className="navbar-brand"
    href="#home"
  >
     Credit Card System 
  </a>
</nav>
`;
